lib/classes/PluginManager.js

```diff
@@ -215,6 +223,47 @@ class PluginManager {
     );
   }
 
+  getLocalPluginsPaths() {
+    const fs = require('fs');
+
+    const pluginsPaths = this.localPluginsPaths.map(({entryFileRealPath, originalPath}) => {
+      if (!entryFileRealPath) {
+        return originalPath;
+      }
+
+      // Scenario 1: a file was specified WITH extension, eg "./folder/my-plugin.js"
+      if (originalPath.endsWith('.js')) {
+        return entryFileRealPath;
+      }
+
+      let realPath;
+      try {
+        realPath = fs.realpathSync(originalPath);
+      } catch (err) {
+        realPath = fs.realpathSync(`${originalPath}.js`);
+      }
+
+      // Scenario 2: a file was specified WITHOUT extension, eg "./folder/my-plugin" (where "my-plugin.js" is a file)
+      if (entryFileRealPath === realPath) {
+        return entryFileRealPath;
+      } else if (path.dirname(entryFileRealPath) === realPath) {
+        // Scenario 3: a folder was specified, eg "./folder/my-plugin" (where "my-plugin" is a package folder)
+
+        // Ensure folders end with '/' to distinguish from files (autocorrects to '\' on Windows)
+        return path.join(path.dirname(entryFileRealPath), '/');
+      }
+
+
+      return null;
+    });
+
+    return pluginsPaths;
+  }
+
+  recordLocalPluginPath(entryFileRealPath, originalPath) {
```

Comment:
Let's not create such methods (there's no real value in creating alias for `this.localPluginsPaths.push`